cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -439,6 +443,110 @@ func (dc *DownstreamController) syncEndpoints() {
 	}
 }
 
+func (dc *DownstreamController) syncPersistentvolumes() {
+	var operation string
+	for {
+		select {
+		case <-beehiveContext.Done():
+			klog.Warning("Stop edgecontroller downstream syncPersistentvolumes loop")
+			return
+		case e := <-dc.persistentvolumsManager.Events():
+			pv, ok := e.Object.(*v1.PersistentVolume)
+			if !ok {
+				klog.Warningf("Object type: %T unsupported", pv)
+				continue
+			}
+
+			switch e.Type {
+			case watch.Added:
+				operation = model.InsertOperation
+			case watch.Modified:
+				operation = model.UpdateOperation
+			case watch.Deleted:
+				operation = model.DeleteOperation
+			default:
+				// unsupported operation, no need to send to any node
+				klog.Warningf("persistentVolume event type: %s unsupported", e.Type)
+				continue
+			}
+
+			nodes := dc.lc.PersistentvolumNodes(pv.Namespace, pv.Name)
```

Comment:
it seems the `PersistentvolumNodes` has not been populated.

Reply:
> it seems the `PersistentvolumNodes` has not been populated.

location.go have content.